Slim/Player/Protocols/HTTP.pm

```diff
@@ -1029,7 +1030,9 @@ sub getMetadataFor {
 	# Check for parsed WMA metadata, this is here because WMA may
 	# use HTTP protocol handler
 	my $song = $client->playingSong();
-	if ( $song && $song->track->url eq $url ) {
+	my $current = $song->currentTrack->url eq $url if $song;
+
+	if ( $current ) {
 		if ( my $meta = $song->pluginData('wmaMeta') ) {
```

Comment:
I have a plugin which uses `wmaMeta` to override the metadata shown. This no longer work, as `$current` is not true. Where shall I start the investigation?

Reply:
I've updated the PR with what I think is the solution: need to check for the track and currentTrack as well

Reply:
Which PR?

Reply:
Oh, the new one, of course.